configure.ac

```diff
@@ -174,6 +174,8 @@ if [test $HAVE_SNIFFER = yes]; then
 fi
 
 if [test $IS_FREEBSD = no]; then
+	AC_CHECK_HEADERS([pci/pci.h],,[AC_MSG_ERROR([pciutils header files not found])])
+	AC_CHECK_LIB([pci], [pci_init], [LIBPCI=-lpci], AC_MSG_ERROR([libpci not found]))
```

Comment:
please add 
LIBS=$LIBS" -lpci"

as currently it doesn't compile with "./autogen.sh ; ./configure ; make -j"

Reply:
fixed. thanks